services/pkgreview/package-rating.service.js

```diff
@@ -0,0 +1,80 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { BaseJsonService } = require('..')
+
+const schema = Joi.object({
+  name: Joi.string().required(),
+  type: Joi.string().required(),
+  icon: Joi.string().required(),
+  starString: [Joi.string().optional(), Joi.allow(null)],
+  reviewsCount: Joi.number(),
+  rating: [Joi.number().optional(), Joi.allow(null)],
+}).required()
+
+module.exports = class PkgreviewRating extends BaseJsonService {
+  static get category() {
+    return 'rating'
+  }
+
+  static get route() {
+    return {
+      base: 'pkgreview/rating',
+      pattern: ':pkgManager/:pkgSlug+',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'pkgreview.dev Package Ratings',
+        namedParams: { pkgManager: 'npm', pkgSlug: 'react' },
+        staticPreview: {
+          label: 'pkgreview.dev',
+          message: '4/5 (125)',
+          color: '#4F78FE',
+        },
+      },
+    ]
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'pkgreview.dev',
```

Comment:
Remember that our labels need to be non-promotional and reflect the data point they are conveying. This should be
```suggestion
      label: 'rating',
```

Reply:
Sure! Doing it!